service/src/main/java/cleveres/tricky/cleverestech/KeystoreInterceptor.kt

```
@@ -324,10 +331,15 @@ object KeystoreInterceptor : BinderInterceptor() {
                 null
             }
 
-        // Keep the service-level Binder transparent for getSecurityLevel. StrongBox is neither
-        // acquired nor registered with our child interceptor, so StrongBox binder identity and
-        // generateKey replies stay entirely inside the platform. The root service only needs
-        // getKeyEntry for cached TEE replacement-chain handling and provenance-safe passthrough.
+        val strongbox =
+            try {
+                ks.getSecurityLevel(SecurityLevel.STRONGBOX)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)